clickhouse-admin/types/src/config.rs

```diff
@@ -137,6 +137,13 @@ impl ReplicaConfig {
         </default>
     </quotas>
 
+    <query_log>
+        <database>system</database>
+        <table>query_log</table>
+        <engine>Engine = MergeTree ORDER BY event_time TTL event_date + INTERVAL 7 DAY</engine>
```

Comment:
Would this need to be `ReplicatedMergeTree`?

Reply:
I don't think we gain much by replicating this data. The table already includes queries that are run internally between the nodes to handle an external query that needs replicated data. And the point of these logs is to help debug performance issues tied to a particular ClickHouse replica -- if there's some hardware failure that causes the replica to fail, we won't be getting any queries from it anyway.